cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -26,31 +31,39 @@
     protected static final String REST_API_PORT_NAME = "rest-api";
 
     private static final String NAME_SUFFIX = "-connect";
+    private static final String METRICS_CONFIG_SUFFIX = NAME_SUFFIX + "-metrics-config";
 
     // Configuration defaults
     protected static final String DEFAULT_IMAGE =
             System.getenv().getOrDefault("STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE", "strimzi/kafka-connect:latest");
     protected static final int DEFAULT_REPLICAS = 3;
     protected static final int DEFAULT_HEALTHCHECK_DELAY = 60;
     protected static final int DEFAULT_HEALTHCHECK_TIMEOUT = 5;
+    protected static final boolean DEFAULT_KAFKA_CONNECT_METRICS_ENABLED = false;
 
     // Configuration keys (in ConfigMap)
     public static final String KEY_IMAGE = "image";
     public static final String KEY_REPLICAS = "nodes";
     public static final String KEY_HEALTHCHECK_DELAY = "healthcheck-delay";
     public static final String KEY_HEALTHCHECK_TIMEOUT = "healthcheck-timeout";
+    public static final String KEY_METRICS_CONFIG = "connect-metrics-config";
```

Comment:
All of these options are named without any prefix. Only this one now starts with *connect-*. Wouldn't it be better to name it just `metrics-config`?